### ISSUE TITLE
Single-sourcing to RAP

### ISSUE THREAD

**Issue by fsteeg** [spinfo/drc#199]
Currently blocked by:
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=262787
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=250697
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=318900

**Comment by fsteeg** [comment]
Duplicate of #97
